EVA Website

EDGAhab opened the pull request.

gaurav274 commented:
@EDGAhab Why do we have the entire eva repository inside the website?
And is it possible to directly use the Readme on GitHub rather than making a copy inside the website folder? If not, please update the readme. Thanks!

jarulraj commented:
Can we get rid of the eva source code?

jarulraj commented on _config.yml:
```diff
@@ -0,0 +1,60 @@
+baseurl: "/eva" # the subpath of your site, e.g. /blog
+url: "https://georgia-tech-db.github.io" # the base hostname & protocol for your site, e.g. http://example.com
+
+show_downloads: true
```
We can disable this option.

EDGAhab replied:
Okay! Disabled

EDGAhab commented:
> Can we get rid of the eva source code?

I just got rid of the source code!

EDGAhab commented:
> @EDGAhab Why do we have the entire eva repository inside the website? And is it possible to directly use the Readme on GitHub rather than making a copy inside the website folder? If not, please update the readme. Thanks!

I deleted the eva repo and updated the readme.md!
